### ISSUE TITLE
deleting sub wallet

### ISSUE THREAD

**Issue by Sem-68** [Berry-Pool/nami-wallet#478]
If i delete a sub wallet with assets in it will the asset go to the main wallet?

**Comment by alessandrokonrad** [comment]
No, they will stay in the sub-wallet. You can easily recover the sub-wallet by creating a new account again. The order of your accounts is always the same

**Comment by Sem-68** [comment]
I have restored the nami wallet on ccvault but the assets from sub wallets aren't there.  Sorry but I'm having troubles getting my head around the sub wallet thing.  How do I open these sub wallets in ccvault?
